test/BUILD.gn

```diff
@@ -241,7 +272,6 @@ test("brave_unit_tests") {
       "//brave/chromium_src/components/search_engines/brave_template_url_prepopulate_data_unittest.cc",
       "//brave/chromium_src/components/search_engines/brave_template_url_service_util_unittest.cc",
       "//brave/chromium_src/chrome/browser/external_protocol/external_protocol_handler_unittest.cc",
-      "//brave/chromium_src/components/translate/core/browser/translate_manager_unittest.cc",
```

Comment:
I rename this file to brave_translate_manager_unittest.cc, then make "components/translate/core/browser:unit_tests" target build because I couldn't add "components/translate/core/browser:unit_tests"  to the "brave/test:brave_unit_tests" target. If anyone know a better way to fix this dependency issue, please let me know.